host-configuration/docs/2_AGL_Application_Framework.md

```diff
@@ -0,0 +1,24 @@
+# AGL Application Framework
+
+## AGL application framework for debian
+
+```bash
+sudo apt-get install agl-app-framework-binder
+```
+
+## AGL application framework for openSUSE
+
+```bash
+sudo zypper install agl-app-framework-binder
+```
+
+## AGL application framework for fedora
+
+```bash
+sudo dnf install agl-app-framework-binder
+```
+
+## AGL application framework documentation
+
+You can find the AGL application framework documentation
+ [here](/docs/apis_services/en/dev/reference/af-main/0-introduction.html).
```

Comment:
This link is not usable when to generate a PDF.
Maybe there is a better solution but the only way I know for now is to use full instead of partial link :  
  http://docs.automotivelinux.org/docs/apis_services/en/dev/reference/af-main/0-introduction.html